playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -25,6 +25,14 @@
       cluster: "{{ cluster_id }}"
       type: "{{ k8s_type }}"
 
+  - set_fact:
+      a_master: "{{ master_names[0] }}"
+  - add_host: name={{ a_master }} groups=service_master
+
 - include: update.yml
 
+- include: ../../common/openshift-cluster/create_services.yml
+  vars:
+     create_services__master: "{{ service_master }}"
```

Comment:
Under that current [style guide](https://github.com/openshift/openshift-ansible/blob/master/docs/style_guide.adoc#ansible-role-variables), this variable should something like (since it's a 2 word thing, you have some leeway):
```
csvc_master: "{{ service_master }}"
```